common-libs/common/src/_baseDocumentLoader/localContexts.ts

```diff
@@ -236,4 +236,41 @@ export const localContexts: Record<string, Record<string, any>> = {
       proof: { '@id': 'https://w3id.org/security#proof', '@type': '@id', '@container': '@graph' },
     },
   },
+  'https://w3id.org/vc-revocation-list-2020/v1': {
```

Comment:
@Volland could you please update the change log - to describe that performance for revocable VC was improved + documentation updates + minor fixes (as confirmSignup with issue key stone VC option)